openshift/templates/db-backup/mongo/db-backup.dc.json

```diff
@@ -0,0 +1,590 @@
+      "required": true,
+      "value": "database-user"
+    },
+    {
+      "name": "POSTGRES_PASSWORD_KEY_NAME",
+      "displayName": "Database Password Key Name",
+      "description": "The database password key name stored in database deployment resources specified by POSTGRES_DEPLOYMENT_NAME.",
+      "required": true,
+      "value": "database-password"
+    },
+    {
+      "name": "MONGO_DEPLOYMENT_NAME",
+      "displayName": "Database Deployment Name",
+      "description": "The name associated to the database deployment resources.  In particular, this is used to wire up the credentials associated to the database.",
+      "required": true,
+      "value": "mongo"
+    },
+    {
+      "name": "MONGO_USER_KEY_NAME",
+      "displayName": "Database User Key Name",
+      "description": "The datbase user key name stoed in database deployment resources specified by POSTGRES_DEPLOYMENT_NAME.",
+      "required": true,
+      "value": "database-user"
+    },
+    {
+      "name": "MONGO_PASSWORD_KEY_NAME",
+      "displayName": "Database Password Key Name",
+      "description": "The database password key name stored in database deployment resources specified by POSTGRES_DEPLOYMENT_NAME.",
+      "required": true,
+      "value": "database-password"
+    },
+    {
+      "name": "MSSQL_SA_PASSWORD",
+      "displayName": "MSSQL SA Password",
+      "description": "The database password to use for the local backup database.",
+      "required": false
+    },
+    {
+      "name": "TABLE_SCHEMA",
+      "displayName": "Table Schema",
+      "description": "The table schema for your database.  Used for Postgres backups.",
+      "required": true,
+      "value": "public"
+    },
+    {
+      "name": "BACKUP_STRATEGY",
+      "displayName": "Backup Strategy",
+      "description": "The strategy to use for backups; for example daily, or rolling.",
+      "required": true,
+      "value": "rolling"
+    },
+    {
+      "name": "FTP_SECRET_KEY",
+      "displayName": "FTP Secret Key",
+      "description": "The FTP secret key is used to wire up the credentials associated to the FTP.",
+      "required": false,
+      "value": "ftp-secret"
+    },
+    {
+      "name": "FTP_URL",
+      "displayName": "FTP Server URL",
+      "description": "The URL of the backup FTP server",
+      "required": false,
+      "value": ""
+    },
+    {
+      "name": "FTP_USER",
+      "displayName": "FTP user name",
+      "description": "FTP user name",
+      "required": false,
+      "value": ""
+    },
+    {
+      "name": "FTP_PASSWORD",
+      "displayName": "FTP password",
+      "description": "FTP password",
+      "required": false,
+      "value": ""
+    },
+    {
+      "name": "WEBHOOK_URL",
+      "displayName": "Webhook URL",
+      "description": "The URL of the webhook to use for notifications.  If not specified, the webhook integration feature is disabled.",
+      "required": false,
+      "value": ""
```

Comment:
We should probably set this up too

Reply:
will make it a secret ref

Reply:
template already has this url setup as a secret